spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/pom.xml

```diff
@@ -90,6 +90,74 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
       </plugin>
+
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>3.3.1</version>
+        <dependencies>
+          <dependency>
+            <groupId>com.google.cloud.tools</groupId>
+            <artifactId>jib-native-image-extension-maven</artifactId>
+            <version>0.1.0</version>
+          </dependency>
+        </dependencies>
+        <configuration>
+          <from>
+            <image>debian:bookworm-20221219-slim</image>
+          </from>
+          <to>
+            <image>gcr.io/mpeddada-test/secretmanager-native-image3:latest</image>
```

Comment:
This is currently pointing to a private container registry endpoint. This needs to be replaced with the desired destination.